inc/compat/plugins/compat-plugin-woocommerce-germanized.php

```diff
@@ -81,6 +79,19 @@ public function late_hooks() {
 		add_filter( 'fc_substep_text_billing_address_field_keys_skip_list', array( $this, 'change_substep_text_extra_fields_skip_list_billing' ), 10 );
 	}
 
+	/*
+	* Add or remove checkout page hooks.
+	*/
+	public function checkout_hooks() {
+		// Bail if not on checkout page
+		if ( ! FluidCheckout_Steps::instance()->is_checkout_page_or_fragment() ) { return; }
```

Comment:
@vlad-dedikov, this change is causing the "Place order" button to be displayed apparently in the wrong position on the checkout page.

- Why was it necessary to move the code related to the place order button position to this separare function and check for whether running on the checkout page?